app/design/frontend/Jungle-luma/jungle_theme/etc/view.xml

```diff
@@ -0,0 +1,314 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
```

Comment:
When you copy default magento files, coupyright comment block should be removed, remove it please

Reply:
Removed the comment block.